packages/documentation/package.json

```diff
@@ -83,7 +83,8 @@
     "storybook": "8.4.7",
     "typescript": "5.8.2",
     "typescript-eslint": "8.20.0",
-    "vite": "6.2.6"
+    "vite": "6.1.3",
```

Comment:
Is there a special reason, why you downgrade vite here?
If not, update it to "6.2.6" as it was on "main"